src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -343,6 +358,10 @@ const PopularRepositoriesTable = () => {
 
   const countIsZero = !data?.data?.length;
 
+  const clearFilters = () =>
+    setFilterData({ searchQuery: '', versions: [], arches: [], statuses: [] });
+  setSearchValue('');
```

Comment:
what does this line of code do.

Reply:
I added that from the filters on the Content List Filters following the pattern but realized what you told me earlier

Reply:
This line of code is not a part of the "clearFilters" function above. It is separate. 
What does this line of code do by itself here in such a case?

Reply:
I removed the line it worked